python/ray/autoscaler/_private/load_metrics.py

```diff
@@ -76,6 +76,7 @@ def __init__(self, local_ip=None):
         self.last_heartbeat_time_by_ip = {}
         self.static_resources_by_ip = {}
         self.dynamic_resources_by_ip = {}
+        self.raylet_id_by_ip = {}
         self.resource_load_by_ip = {}
```

Comment:
Do we need raylet id by ip if raylet id is just the ip?

Reply:
raylet id is something other than ip and must be supplied to drain node calls